implicits/src/com/rallyhealth/weepickle/v1/implicits/MacroImplicits.scala

```diff
@@ -341,7 +341,7 @@ object MacroImplicits {
       }
       q"""
         ..${for (arg <- args)
-        yield q"private[this] lazy val ${arg.localTo} = implicitly[${c.prefix}.To[${arg.argType}]]"}
+        yield q"private[this] val ${arg.localTo} = implicitly[${c.prefix}.To[${arg.argType}]]"}
```

Comment:
What if the implicit is summoned in the same scope as the companion object? Is that even possible? If not, I think this is fine.

Reply:
I'm not sure. You could try it.

The tests are currently failing on self-recursive and co-recursive types. For example, this causes `java.lang.StackOverflowError` in the compiler:

```scala
case class IntTree(value: Int, children: List[IntTree])
object IntTree {
  implicit def rw: RW[IntTree] = WeePickle.macroFromTo
}
```

I'm not sure how to solve without the lazy vals or something like it.